scripts/docker-helpers.sh

```diff
@@ -176,7 +176,7 @@ function docker_start() {
 
 function docker_terminate() {
 	echo -n "###   Uninstalling kruize for docker..."
-	docker stop kruize grafana prometheus cadvisor 2>/dev/null
+	docker stop kruize grafana prometheus cadvisor &> /dev/null
```

Comment:
Why is this change needed ? We might run into issues such as [this](https://unix.stackexchange.com/questions/356408/strange-problem-with-trap-and-sigint) if we background the process.

Reply:
@dinogun  Thanks for the review. 

Aah! Sorry i haven't thought about, When i was testing my PR i saw `stdout` of docker stop (which is the name of containers stopped) I felt it a bit unaligned as the `	echo -n "###   Uninstalling kruize for docker..."` doesn't produce an new line and the output looked as 

```
###   Uninstalling kruize for docker...kruize
grafana
prometheus
cadvisor
done
```

So jus backgrounded the process and directed `stdout` as well to `/dev/null`

Will redirecting `stderr` to `stdout` and `stdout` to `/dev/null` be a good solution ?

Reply:
reverted back to previous changes